webpack.config.ts

```diff
@@ -95,7 +99,11 @@ export function getClientConfig({
         */
         {
           test: /\.css$/,
-          include: [path.resolve(__dirname, 'node_modules')],
+          include: [
+            __dirname.endsWith(path.sep + 'dist') // __dirname is `[..]/dist` when built server is run in dev
+              ? path.resolve(__dirname, '..', 'node_modules')
```

Comment:
It probably doesn't make sense for this file to understand this relationship.

How about passing in directories as arguments to these functions (`getServerConfig`/`getClientConfig`)?

We can remove all references to `__dirname` that way.

Reply:
Yeah, that's a better approach. I'll update this PR tomorrow.